docs/specs/api.md

```diff
@@ -0,0 +1,56 @@
+# API
+
+Here are the endpoints that are not part of OIDC, but essential to provide some features of Authgear.
+
+## /api/identity/biometric/remove
+
+This endpoint is for removing Biometric identity.
+The user still can remove biometric identity in the settings page.
```

Comment:
> Is it because all of them implement "Biometric" login like storing refresh/access token locally, and it is actually not a login, but a single session?

For HSBC, I guess they use biometric authentication to retrieve the mobile security key (a password) in the device keychain. So it should still be a new login every time biometric authentication is used.

> Or we just store the biometrics login data at server and ignore it...?

Do you mean we keep the current design and just do not show the biometric identity in the settings page?

> Or make it optional, so advanced users can also disable biometric login per device?

Not sure what you meant.

Reply:
I meant, if we keep the current design, make it optional in setting page to show or hide the biometric identities.

It seems uncommon, but in our design it seems necessary 

(e.g. A potential security breach is someone stole the accounts, then create a Biometric login by API, but just store the key pair themselves, in this case the hacker have a way to always come back to the account without the user knowing it. Show list it in session page seems useful.)

Reply:
I prefer we just show the biometric identities along with other identities (login IDs, OAuth), until we learn a strong reason from the developer not to do so.

Reply:
@louischan-oursky UX-wise, I don't even think it should be shown along with other identities. It should be shown in the sessions part?

I prefer to make it hidden by default, and let admin to enable it, just try to follows a "normal" behaviour before we know we should do differently otherwise.